db/version_edit_handler.cc

```diff
@@ -564,12 +564,13 @@ Status VersionEditHandler::LoadTables(ColumnFamilyData* cfd,
   assert(builder_iter->second != nullptr);
   VersionBuilder* builder = builder_iter->second->version_builder();
   assert(builder);
+  const MutableCFOptions* moptions = cfd->GetLatestMutableCFOptions();
   Status s = builder->LoadTableHandlers(
```

Comment:
Just so you know : potential merge conflict with https://github.com/facebook/rocksdb/pull/11288